astropy/constants/constant.py

```diff
@@ -9,12 +9,11 @@
 from astropy.units.quantity import Quantity
 from astropy.utils import lazyproperty
 from astropy.utils.exceptions import AstropyUserWarning
-from astropy.utils.misc import InheritDocstrings
 
 __all__ = ['Constant', 'EMConstant']
 
 
-class ConstantMeta(InheritDocstrings):
+class ConstantMeta(type):
```

Comment:
Is this correct, @stargaser ?

Reply:
From inspecting the code here and in the `InheritDocstrings` class, it looks correct to me.